doc/contributor/howto-guide-declare-a-library-ga.md

```diff
@@ -116,5 +116,9 @@ quickstart guide to reference the rules and targets without an `experimental-`
 prefix.  In the following release you can remove these rules and targets.
 
 ```shell
-for d in "${ga[@]}"; do sed -i 's/experimental-//' google/cloud/$d/quickstart/BUILD.bazel; done
+for d in "${ga[@]}"; do
+  sed -i 's/experimental-//' \
+    "google/cloud/${d}/quickstart/CMakeLists.txt" \
```

Comment:
You can remove the `experimental-` as soon as renovate bot updates our builds to use a release with the GA version of the library.  For CMake, we recommend `vcpkg` in the instructions, we should wait until our GA libraries are available there.  The two events are not simultaneous, but often happen almost at the same time.